webpack.config.release.js

```diff
@@ -22,7 +22,7 @@ const settings = require('./tasks/helper')
 module.exports = {
   // Entry point for static analyzer:
   entry: {
-    bundle: './app/main.js'
+    bundle: ['babel-polyfill', './app/main.js']
```

Comment:
Could we narrow this down abit. What is the actuall polyfill that we are missing? I imagine its something to do with arrays.

Reply:
I could probably be more granular and detect which es6 functions aren't supported by IE11 and Edge. Let me look at it